.github/styles/README.md

```diff
@@ -0,0 +1,30 @@
+# Style Guide
+The team follows the Google developer's and the Microsoft's documentation style guide. The style rules are configured with the help of [Vale](https://vale.sh/) linter.
+
+## Prerequisites
+Please ensure that the below prerequisites are met before you configure vale linter. You should have:
+
+* [Brew](https://brew.sh) installed on your Mac, or [Chocolatey](https://chocolatey.org/) installed on your Windows machine.
+   > For windows, you could also use the command prompt and curl command to install brew - `curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh`
+* Vale installed on your mac/window. 
+
+> If not already installed, you could install vale using `brew install vale` for mac, or `choco install vale` on Windows. Once installed, verify the version ` vale -v`. You can also check the [Vale installation page](https://vale.sh/docs/vale-cli/installation/) for other options.
+
+* Install the [vale extension](https://marketplace.visualstudio.com/items?itemName=errata-ai.vale-server) for VS Code Editor.
+
+> You could also use [vale cli](https://vale.sh/docs/vale-cli/overview/). 
+
+## Setup
+Follow the below steps to setup the linter:
+* Clone the documentation [appsmith-docs](https://github.com/appsmithorg/appsmith-docs) repository and open it in VS Code Editor. 
+* Create a branch that you need to work on. 
+* Configure the vale extension to read the `.vale.ini` file. Go to Extension Settings --> Select `Workspace` tab --> Set up the path of `.vale.ini` file for `Vale › Vale CLI: Config` parameter. 
```

Comment:
Can we make this a numbered list please? If I understand right, they are steps to be taken in order, right? Numbered lists are perfect for that. 🙂

```suggestion
1. Clone the documentation [appsmith-docs](https://github.com/appsmithorg/appsmith-docs) repository and open it in VS Code Editor. 
2. Create a branch that you need to work on. 
3. Configure the vale extension to read the `.vale.ini` file. Go to Extension Settings --> Select `Workspace` tab --> Set up the path of `.vale.ini` file for `Vale › Vale CLI: Config` parameter. 
```

Quoting Google's Technical Writing Course at https://developers.google.com/tech-writing/one/lists-and-tables:

> Use a bulleted list for unordered items; use a numbered list for ordered items.